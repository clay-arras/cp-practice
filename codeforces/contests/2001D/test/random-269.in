1
17
3 3 4 21 4 6 2 1 15 11 22 13 7 3 22 6 4
