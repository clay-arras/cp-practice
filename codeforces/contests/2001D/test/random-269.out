11
21 2 1 15 11 13 7 3 22 6 4
